src/components/molecules/card/card.html

```diff
@@ -0,0 +1,11 @@
+<div class="card">
+  <p class="card-title">card-title</p>
+  <div class="vector"></div>
+  <figure class="card-img">
+    <img src="../../../img/card-1.jpg" class="card-image" />
+  </figure>
```

Comment:
figure 要素を使えているの、理解度が高くて最高ですね！

card-img　と　card-image　はほとんど同じ意味で、クラス名としてどっちがどっちか分かりづらいので、下のように直してもいいと思います。

```suggestion
  <figure class="card-image-wrap">
    <img src="../../../img/card-1.jpg" class="card-image" />
  </figure>
```